paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -216,6 +251,43 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         }
     }
 
+    private fun setupHeader() {
+        header.apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                val text = viewModel.headerText.collectAsState(null)
+                text.value?.let {
+                    StripeTheme {
```

Comment:
I'm starting to think an extension method (maybe `setStripeContent`) would be nice to handle the theme setting in a central place. I can pick this up as a follow up if you think it's a good idea.

Reply:
I don’t think we need that. Once `PaymentSheetActivity` is in Compose, we’ll just have to wrap it once at the root.